be/src/util/stack_util.cpp

```diff
@@ -223,16 +253,27 @@ std::string get_stack_trace_for_threads_with_pattern(const std::vector<int>& tid
         ret += stack_trace;
         ret += "\n";
     }
-    ret += strings::Substitute("total $0 threads, $1 identical groups", tids.size(), task_map.size());
+    int64_t avg_task_cost_us = 0;
+    if (task_done_count == 0) {
+        min_task_cost_us = 0;
+        max_task_cost_us = 0;
+    } else {
+        avg_task_cost_us = total_task_cost_us / task_done_count;
+    }
+    ret += fmt::format(
+            "{}total {} threads, {} identical groups, finish {} threads, total cost {} us, symbolize cost {} us,"
+            " thread block(avg/min/max) {}/{}/{} us, ",
+            line_prefix, tids.size(), task_map.size(), task_done_count, (MonotonicMicros() - start_us),
+            total_symbolize_cost_us, avg_task_cost_us, min_task_cost_us, max_task_cost_us);
```

Comment:
Why is it necessary to obtain the `cost` here?

Reply:
just want to know how much impact the stack trace has